test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -107,6 +107,19 @@ describe('Dropdown Component', () => {
     dropdownMenuIsClosed()
   })
 
+  it('blurs the Dropdown node on close', () => {
+    wrapperMount(<Dropdown options={options} selection defaultOpen />)
+
+    dropdownMenuIsOpen()
+    wrapper.simulate('click')
+    dropdownMenuIsClosed()
+
+    const instance = wrapper.instance()
+    sandbox.spy(instance._dropdown, 'blur')
```

Comment:
So close, just need to start spying _before_ closing the Dropdown.  The spy will track all activity after it is initialized.  I just moved the `const instance` part above the `dropdownMenuIs..` part:

```jsx
wrapperMount(<Dropdown options={options} selection defaultOpen />)

const instance = wrapper.instance()
sandbox.spy(instance._dropdown, 'blur')

dropdownMenuIsOpen()
wrapper.simulate('click')
dropdownMenuIsClosed()

instance._dropdown.blur
  .should.have.been.calledOnce()
```